src/test/java/competition/subsystems/armlifting/CollectorArmLiftingSubsystemTest.java

```diff
@@ -0,0 +1,47 @@
+package competition.subsystems.armlifting;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import competition.BaseCompetitionTest;
+import competition.subsystems.armlifting.CollectorArmLiftingSubsystem;
+
+public class CollectorArmLiftingSubsystemTest extends BaseCompetitionTest {
+    @Test
+    public void testCollectorArmLiftingUp() {
+        CollectorArmLiftingSubsystem collectArmLifting = this.injector.getInstance(CollectorArmLiftingSubsystem.class);
+        collectArmLifting.up();
+        assertEquals(1, collectArmLifting.liftingCollectorArmMotor.getMotorOutputPercent(), 0.001);
+    }
+    @Test
+    public void testCollectorArmLiftingFrozen() {
+        CollectorArmLiftingSubsystem collectArmLifting = this.injector.getInstance(CollectorArmLiftingSubsystem.class);
+        collectArmLifting.frozen();
+        assertEquals(0, collectArmLifting.liftingCollectorArmMotor.getMotorOutputPercent(), 0.001);
+    }
+
+    @Test
+    public void testCollectorArmLiftingIsAtMaximum() {
+        CollectorArmLiftingSubsystem collectArmLifting = this.injector.getInstance(CollectorArmLiftingSubsystem.class);
+        collectArmLifting.setCurrentArmLiftHeight(1);
+        collectArmLifting.setPower(1);
+        assertEquals(0, collectArmLifting.liftingCollectorArmMotor.getMotorOutputPercent(), 0.001);
+    }
+    
+    @Test
+    public void testCollectorArmLiftingIsAtMinimum() {
+        CollectorArmLiftingSubsystem collectArmLifting = this.injector.getInstance(CollectorArmLiftingSubsystem.class);
+        collectArmLifting.setCurrentArmLiftHeight(-1);
+        collectArmLifting.setPower(-1);
+        assertEquals(0, collectArmLifting.liftingCollectorArmMotor.getMotorOutputPercent(), 0.001);
+
+    }
+
+    @Test
+    public void testCollectorArmLiftingStop() {
```

Comment:
This doesn't actually test that the system stops - you need to have it moving first to confirm that it will stop.